src/lib/storage/table.hpp

```diff
@@ -205,6 +205,13 @@ class Table : private Noncopyable {
    */
   size_t memory_usage(const MemoryUsageCalculationMode mode) const;
 
+  /**
+   * If the table is value clustered in any way, the ColumnID of the segment by which it is clustered
+   * will be returned.
```

Comment:
The explanation is quite technical. You explain why you may end up with a value-clustered table and dive deep into bit representations. I wonder if this is helpful for people who are just getting started with the Table interface.

Suggestion: "Tables may be clustered by one or more columns. Each value within such a column will occur in exactly one chunk. This information can be used, e.g., when executing a GROUP BY on a clustered column, which can then look at individual chunks instead of the entire table. Note that this does not necessarily mean that the data in the column is ordered. At the same time, ordered data is not necessarily clustered as a value could still occur at the end of one chunk and the beginning of the next."

Reply:
Added your suggestion almost only additions :)

Reply:
Typos: exaclty, another (not: an other), missing full stop after next, the ColumnID**s** will be returned